specification/assets/gherkin/metadata.feature

```diff
@@ -0,0 +1,27 @@
+@Metadata
+Feature: Metadata
+
+  Background:
+    Given a provider is registered
```

Comment:
```suggestion
    Given a stable provider
```

to be more in sync with our new flagd gherkin files, which have been already an inspiration for this file